deploy.rb

```diff
@@ -17,8 +17,8 @@
 
 def update_repo(repo_dir)
   Dir.chdir(repo_dir) do
-    `git checkout config/deploy.rb`
-    `git checkout master 2> /dev/null && git pull`
+    `git fetch origin`
+    `git reset --hard $(git symbolic-ref refs/remotes/origin/HEAD) 2> /dev/null`
```

Comment:
This would be worth testing before merging. happy to work with @aaron-collier this week on this.